fxa-auth-db-server/docs/API.md

```diff
@@ -59,7 +59,8 @@ The following datatypes are used throughout this document:
     * sessions                  : `GET /account/:id/sessions`
     * devices                   : `GET /account/:id/devices`
 * Devices:
-    * upsertDevice              : `PUT /account/:id/device/:deviceId`
+    * createDevice              : `PUT /account/:id/device/:deviceId`
+    * updateDevice              : `POST /account/:id/device/:deviceId`
```

Comment:
I'm not thrilled about using `PUT` and `POST` on the same end-point with such different semantics.  We seem to have a convention of using `PUT /blah/:id` for creation and `POST /blah/:id/update` for updates on e.g. some of the token types, which we should consider just copying here for consistency.